src/openforms/appointments/tests/test_views.py

```diff
@@ -20,13 +20,14 @@
     SubmissionFactory,
     SubmissionStepFactory,
 )
+from openforms.tests.mixins import FrontendRedirectMixin
```

Comment:
Test helpers go in `openforms.utils.tests`, see the documentation: https://open-forms.readthedocs.io/en/stable/developers/backend/core/testing-tools.html

In this particular case I'd rather put them in `openforms.frontend.tests` even and import from there & this documentation page needs to be updated to mention the test helper.